docs/user_authentication/third_party_token_management.md

```diff
@@ -0,0 +1,25 @@
+
+```sequence {theme: 'simple'}
+Title: Third party idp token management
+Pbench-Dashboard->Pbench-Server:[1] GET login page
+Pbench-Server->Pbench-Dashboard:[2] 200 login page
+Pbench-Dashboard->Pbench-Dashboard:[3] Generate Authorization request
+Pbench-Dashboard->Identity-Provider:[4] POST Authorization request
+Identity-Provider->Identity-Provider:[5] 200 Authenticate User
+Identity-Provider->Identity-Provider:[6] GET consent
+Identity-Provider->Pbench-Dashboard:[7] 200 Authorization code
+Pbench-Dashboard->Identity-Provider:[8] POST Token request
+Identity-Provider->Pbench-Dashboard:[9] 200 Access/Refresh Token payload
+Pbench-Dashboard->Pbench-Server:[10] Send Access_token to get Pbench token
```

Comment:
If the Pbench Server will have an API endpoint for this, it might be worth adding it to the message, and giving the "access token" an ID since we'll want to identify which access token value when it changes.  So perhaps something like:

```
// Here the dashboard sends the newly acquired access token to the Pbench Server to start the token exchange process.
Pbench-Dashboard->Pbench-Server: POST /api/v1/access request (Bearer: Access Token 1)
```